test/jasmine/assets/custom_matchers.js

```diff
@@ -12,7 +12,10 @@ module.exports = {
                     return Math.abs(expected[i] - element) < precision;
                 });
 
-                var passed = tested.indexOf(false) < 0;
+                var passed = (
+                    expected.length === actual.length &&
```

Comment:
`toBeCloseArray` loops over the `actual` array which led to false positive assertions when the expected array is longer than the actual.

The patch fixes this problem.